### PR TITLE
draft of FP101

### PR DESCRIPTION
**Pull request by mcgoodman**
Wrote a draft of FP101 - choosing a study for replication. Includes study guidelines, where to look for studies, and some suggested studies.

### CODE REVIEW

File: FP101.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# FP101: Choose study for replication
+
+
+
+## Study Guidelines
+
+Choose a study that you will replicate a figure or analysis from. A good study (and the corresponding figure or analysis) for replication will meet the following criteria:
+
+- The requisite data, if any are needed, are available online and in a form which does not require significant pre-processing. If the code are available online, that's even better! If you need any data,
```
Comment (FlukeAndFeather) [inline FP101.md]: What's the end supposed to be?

**Review by FlukeAndFeather — CHANGES_REQUESTED** [review]
Looks great Maurice - thanks! There's one incomplete sentence. Otherwise perfect. In your next commit, include the text "Fix #2", which will automatically link your changes to that issue.

**Comment by mcgoodman** [comment]
Removed the sentence fragment! Not really sure what I had intended it to say but reading over it, it seems like it would have been extraneous anyway.

**Review by FlukeAndFeather — APPROVED** [review]
Thanks Maurice!

**Pull request merged**